runtime/util/shchelp_j9.c

```diff
@@ -113,7 +113,7 @@ getLastDollarSignOfLambdaClassName(const char *className, UDATA classNameLength)
 #endif /* JAVA_SPEC_VERSION < 21 */
 
 BOOLEAN
-isLambdaClassName(const char *className, UDATA classNameLength)
+isLambdaClassName(const char *className, UDATA classNameLength, UDATA *prefixLength)
```

Comment:
`prefixLength` perhaps isn't as clear as it could be; maybe something like `deterministicPrefixLength` or `deterministicLength` would be clearer. This is just a nit though, since you have described that it is deterministic in the doxygen.